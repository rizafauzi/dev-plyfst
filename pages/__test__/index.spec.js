import { shallowMount } from '@vue/test-utils'
import index from '../index.vue'

describe('index.vue', () => {
	let wrapper = null

	beforeEach(() => {
		wrapper = shallowMount(index)
	})
	afterEach(() => {
		wrapper.destroy()
	})

	it('should render correctly', () => {
		
		expect(wrapper.element).toMatchSnapshot()
	})

	// it('should change text label when button clicked', () => {
	// wrapper.find('.btn-action').trigger('click.prevent');

	// expect(wrapper.vm.label).toBe('Hello');
	// })
})
