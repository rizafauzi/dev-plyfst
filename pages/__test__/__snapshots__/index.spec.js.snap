// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue should render correctly 1`] = `
<div
  class="container"
>
  <navbar-section-stub />
   
  <showcase-stub />
   
  <footer-section-stub />
</div>
`;
